.github/workflows/mt.yaml

```diff
@@ -53,7 +53,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer update --no-interaction --prefer-dist --no-progress
+          composer install --no-interaction --prefer-dist --no-progress
```

Comment:
I don't remember why do we use "highest" dependencies instead of locked. So lets use locked

Reply:
probably because then we never test infection installed as a package, since it will only picked the locked with the configured platform

Reply:
Sorted out in #1791